packages/cli/src/ui/hooks/useKeypress.ts

```diff
@@ -11,7 +11,81 @@ import { useKeypressContext } from '../contexts/KeypressContext.js';
 export type { Key };
 
 /**
```

Comment:
nit: move these out of useKeypress into a util file for your psuedo-terminal support as these aren't used elsewhere in useKeypress